docs/advanced-setup.md

```diff
@@ -31,3 +31,40 @@ as the watchdog.
 
 !!! Note
     The sd_notify communication between the bot and the systemd service manager will not work if the bot runs in a Docker container.
+
+## Logging to syslog or journald
+
+On many systems the bot can be configured to send its log messages to `syslog` or `journald`. The special values for the `--logfilename` option can be used for this:
```

Comment:
On many *nix systems ... ?

Reply:
Linux is not Unix 😉

Reply:
well then call it linux ... the point here is to be clear that it's not available for windows ...